### PR TITLE
Fix bugs that "copy to clip board" not works

### PR DESCRIPTION
**Pull request by WatanabeToshimitsu**
## What?
Fix bugs that "copy to clip board" not works

## Why?
Bug fixes

## See also
Resolves https://github.com/dataware-tools/dataware-tools/issues/102

## Screenshot or video
FixCopyToClipboard

### CODE REVIEW

**Comment by hdl-service** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/dataware-tools/app-data-browser-next/pull/329#" title="Author self-approved">WatanabeToshimitsu</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=dataware-tools%2Fapp-data-browser-next).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/dataware-tools/app-data-browser-next/blob/master/OWNERS)~~ [WatanabeToshimitsu]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: src/components/organisms/RecordInfo.tsx
Comment (yusukefs) [inline src/components/organisms/RecordInfo.tsx:65]: `console.log` は消し忘れでしょうか？

Comment (WatanabeToshimitsu) [reply src/components/organisms/RecordInfo.tsx]: 指摘あざます！消しました！

**Review by yusukefs — APPROVED** [review]
修正あざます！LGTM

**Comment by hdl-service** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/dataware-tools/app-data-browser-next/pull/329#" title="Author self-approved">WatanabeToshimitsu</a>*, *<a href="https://github.com/dataware-tools/app-data-browser-next/pull/329#pullrequestreview-849897330" title="Approved">yusukefs</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=dataware-tools%2Fapp-data-browser-next).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/dataware-tools/app-data-browser-next/blob/master/OWNERS)~~ [WatanabeToshimitsu,yusukefs]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
